salt/modules/rabbitmq.py

```diff
@@ -111,15 +117,14 @@ def _get_rabbitmq_plugin():
     This works by taking the rabbitmq-server version and looking for where it
     seems to be hidden in /usr/lib.
     '''
-    rabbitmq = __context__.get('rabbitmq-plugins')
+    global RABBITMQ_PLUGINS
 
-    if rabbitmq is None:
+    if RABBITMQ_PLUGINS is None:
         version = __salt__['pkg.version']('rabbitmq-server').split('-')[0]
-        rabbitmq = ('/usr/lib/rabbitmq/lib/rabbitmq_server-{0}'
-                    '/sbin/rabbitmq-plugins').format(version)
-        __context__['rabbitmq-plugins'] = rabbitmq
+        RABBITMQ_PLUGINS = ('/usr/lib/rabbitmq/lib/rabbitmq_server-{0}'
+                            '/sbin/rabbitmq-plugins').format(version)
```

Comment:
I'm fine either way - I'm not part of the core team and won't claim to speak on their behalf. I'm merely making observations as the original author of the module who was curious as to what it looked like now.

While you mention the structure as being in place... it was you who made this structure so I felt justified in discussing its design choices with you on this PR. Had I been pinged on the PR that added Windows functionality, I would have brought it up then instead.

I'll leave my comments as just that. I don't feel strongly enough to make the changes myself and - while I'm not a Windows user myself - I'm really glad to see people taking the time to make Salt work better on the platform.